app/views/configuration/_ui_2.html.haml

```diff
@@ -58,6 +59,19 @@
                   = ui_lookup(:tables => table_view_name)
                 .col-md-8
                   %ul.list-inline= render_view_buttons(resource, @edit[:new][:views][resource])
+          %fieldset
+            %h3
+              = _('VM Visibility')
+            .form-group
+              %label.col-md-8.control-label
+                = _("Set visibility of VMs in trees. Warning: Showing VMs may cause performance issues. So better set it to no.")
```

Comment:
Maybe simplify it instead of extending it?

"Show VMs in Explorer tree. Note: This may make it slow"

(but I like smaller faster things in general)